README.md

```diff
@@ -1 +1,39 @@
-# Lisandro-Seia-s-portfolio
\ No newline at end of file
+# Hello microverse project
+
+> this is my first microverse project
+
+Additional description about the project and its features.
+
+## Built With
+
+- Major languages
+html & css
+- 
```

Comment:
- [ ] OPTIONAL Really nice README you have here well done 👍. you can remove this empty line though 🤗.